assets/scss/03-organisms/_teaser.scss

```diff
@@ -0,0 +1,277 @@
+@import "../00-base/colors";
+@import "../00-base/breakpoints";
+
+$c-gray-dcdcdc: #dcdcdc;
+$c-theme-green: #43956f;
+
+.ma__results-list-container{
+  .ma__general-teaser {
```

Comment:
what's the relationship between Teaser and General Teaser component? Seems like they are using the same class names

Reply:
They just have similar styling needs in this case.